spec/flipper/adapters/mongo_spec.rb

```diff
@@ -16,8 +17,19 @@
 
   before do
     begin
-      collection.drop
-    rescue Mongo::Error::OperationFailure
+      Timeout::timeout(1) do
```

Comment:
Since it truly is a separate concern, I just reverted the commit related to this. Setting the timeout in the mongo client initialization can be a separate PR.